app/reducers/dashboardReducers.js

```diff
@@ -0,0 +1,26 @@
+const job = (state = {}, action) => {
```

Comment:
`jobs` will only be a section on the dashboard page, right? If so, you want `jobs` to just be a property on the broader `dashboard` reducer:

```javascript
import { ADD_JOB } from '../actions/DashboardActionTypes';

const defaultState = {
    jobs: []
};

export default dashboardReducer(state = defaultState, { type, payload }) {
    switch (type) {
        case ADD_JOB:
            return {
                ...state,
                jobs: state.jobs.push({ id: payload.id, companyName: payload.companyName, status: payload.status })
            };
       default:
            return state;
    }
}
```